lib/seahorn/HornifyFunction.cc

```diff
@@ -175,6 +231,14 @@ void SmallHornifyFunction::runOnFunction(Function &F) {
       extractFunctionInfo(BB);
   }
 
+  // If F is an partial function stub, it should not have a body.
+  const FunctionInfo &fi = m_sem.getFunctionInfo(F);
+  if (fi.isInferable) {
+    LOG("seahorn",
+        errs() << "Adding partial fn stub: " << F.getName().str() << "\n");
```

Comment:
the print message says that something will be added, but the code returns without adding anything